package.json

```diff
@@ -105,7 +105,10 @@
       }
     },
     "rules": {
-      "arrow-parens": ["error", "always"],
+      "arrow-parens": [
+        "error",
+        "always"
+      ],
```

Comment:
Whoops, these came from 3.2.1 (master).  i accidentally rebased from master instead of dev.  

Shouldn't dev be rebase from master too?